domains/smpure.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "AchIrsHam",
+           "email": "achmadirsyadhamizan@gmail.com",
+           "discord": "973142398991470602"
+        },
+    
+        "record": {
+            "CNAME": "209.25.142.194:10603"
```

```suggestion
            "A": ["209.25.142.194"]
```
Also it seems to not be accessible